document/FileFormat.mdk

```diff
@@ -898,7 +926,7 @@ the precise version number of the logging component that produced the invalid ou
 
 ### `properties` property { #tool-properties }
 
-A `tool` object may contain a property named `properties`, whose value is property bag.
+A `tool` object may contain a property named `properties` whose value is property bag.
```

Comment:
Wow, there were half a dozen of those. Good catch. All fixed (and added section reference to `&sect;[#property-bags]`)

---
In reply to: [75181111](https://github.com/sarif-standard/sarif-spec/pull/256#discussion_r75181111) [](ancestors = 75181111)